### PR TITLE
Move contributors from README to AUTHORS.txt

### LINKED ISSUE #120: Move contributor list from README.md to AUTHORS.txt
I think we should move the list of contributors in the README file to a dedicated AUTHORS.txt file like for the other repos.

https://github.com/pion/.goassets#contributing

### PR DESCRIPTION
**Pull request by stv0g**
Another try for #132 

Fixes #120

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,19 +26,7 @@ shfmt -i 2 -ci -bn -l -w .
 
 ### Contributing
 
-Check out the **[contributing wiki](https://github.com/pion/webrtc/wiki/Contributing)** to join the group of amazing people making this project possible:
-
-- [Sean DuBois](https://github.com/Sean-Der) - _Original Author of the lint scripts_
-- [Max Hawkins](https://github.com/maxhawkins) - _Git hooks_
```
Comment (at-wat) [inline README.md:32]: Missing in new contributor list

**Review by at-wat — COMMENTED** [review]
As the scripts in this repo was originally committed directly to each repos, early contributors can't be listed from git history.

File: README.md (hunk 1 above)
Comment (stv0g) [reply README.md:32]: I updated the README again to keep Max in the list and also add a link to the automatically generated `AUTHORS.txt`

**Review by at-wat — COMMENTED** [review]
How about https://github.com/pion/.goassets/pull/138?

**Comment by stv0g** [comment]
> How about https://github.com/pion/.goassets/pull/138?

I like it :) I made a minor modification of the message and merged it into my branch.

@at-wat Would you review and/or merge it?

**Review by at-wat — APPROVED** [review]
LGTM

**Comment by stv0g** [comment]
Thanks @at-wat :)

**Pull request merged**